spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/mapping/MappingInterpreterSuite.scala

```diff
@@ -38,6 +38,13 @@ trait MappingInterpreterSuite extends AnyFunSuite with SparkTestBase with Logger
     super.afterAll()
   }
 
+  // todo: connected to #1756 for array-based processing, errCol nullability always comes out false after Spark 3.1 processing
```

Comment:
Is this still an issue?

Reply:
Well, the tests pass now. But this replacement is from `false` to `true`....

Reply:
I think itt should be removed

Reply:
-> https://github.com/AbsaOSS/enceladus/issues/1818